add_book.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
+
+if ($_SERVER['REQUEST_METHOD'] == 'POST') {
+    $title = $_POST['title'];
+    $author = $_POST['author'];
+    $year = $_POST['year'];
+
+    $query = "INSERT INTO books (title, author, year) VALUES ('$title', '$author', $year)";
+    if (mysqli_query($connection, $query)) {
```

Comment:
- Observations: Mixing PHP logic with HTML can lead to maintenance issues and makes code difficult to read.
- Recommendations: Separate the logic from the view using templates or a more structured approach like MVC pattern.

Example:
```php
// Controller
require 'add_book_controller.php';

// Template
require 'views/add_book_form.php';
```